preprocess:
  lidar_type: 2                # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR,
  scan_line: 32
  time_scale: 1e-3             # 兼容不同数据集的时间单位，仅对Velodyne LiDAR(lidar_type=2)生效
  point_filter_num: 10
mapping:
  extrinsic_T: [ 0, 0, 0.28 ]
  extrinsic_R: [ 1, 0, 0,
                 0, 1, 0,
                 0, 0, 1 ]
  registration_type: 1
init:
  init_time_seconds: 2.0               # 静止时间
  init_imu_queue_max_size: 2000        # 初始化IMU队列最大长度
  static_odom_pulse: 5                 # 静止时轮速计输出噪声
  max_static_gyro_var: 0.5             # 静态下陀螺测量方差
  max_static_acce_var: 0.05            # 静态下加计测量方差
  gravity_norm: 9.81                    # 重力大小
  use_speed_for_static_checking: false  # 是否使用odom判断车辆静止

imu:
  num_iterations: 3           # 迭代次数
  quit_eps: 1e-3              # 终止迭代的dx大小
  imu_dt: 0.01                # IMU测量间隔
  gyro_var: 1e-5              # 陀螺测量标准差
  acce_var: 1e-2              # 加计测量标准差
  bias_gyro_var: 1e-6         # 陀螺零偏游走标准差
  bias_acce_var: 1e-4         # 加计零偏游走标准差

gnss:
  gnss_pos_noise: 0.1         # GNSS位置噪声
  gnss_height_noise: 0.1      # GNSS高度噪声
  gnss_ang_noise: 1.0         # GNSS旋转噪声（单位：度）
# imu:
#   bias_gyro_var:
#   bias_acce_var
